README.md

```diff
@@ -29,18 +28,29 @@ client = Notifications::Client.new(service_id, secret_id, base_url)
 client.base_url # => Notifications::Client::PRODUCTION_BASE_URL
 ```
 
-### send sms or email
+Generate an API key by logging in to GOV.UK Notify [GOV.UK Notify](https://www.notifications.service.gov.uk) and going to the API integration page.
+
+You will find your service ID on the API integration page.
 
-send sms or email providing recipient and template
+### Send a message
+
+Text:
 
 ```ruby
-sms = client.send_sms(to: number, template: template_id)
-email = client.send_email(to: number, template: template_id)
-```
+sms = client.send_sms(to: number, 
+                      template: template_id,
+                      personalisation: {
+                      name: "name",
+                      year: "2016"
+  }
+)
```

Comment:
Is this indentation the way they do things in Rubyland? 

I really dislike it because changing the method name means changing the subsequent lines, and you get a messy diff.